.NET/Microsoft.Recognizers.Definitions/English/DateTimeDefinitions.cs

```diff
@@ -541,5 +544,16 @@ public static class DateTimeDefinitions
 			{ "christmaseve", new string[] { "christmaseve" } },
 			{ "newyeareve", new string[] { "newyearseve", "newyeareve" } }
 		};
+		public static readonly Dictionary<string, int> WrittenDecades = new Dictionary<string, int>
+		{
+			{ "twenties", 20 },
+			{ "thirties", 30 },
+			{ "forties", 40 },
+			{ "fifties", 50 },
+			{ "sixties", 60 },
+			{ "seventies", 70 },
+			{ "eighties", 80 },
+			{ "nineties", 90 }

```

Comment:
In UK English the decade of 2000-09 is usually know as "the noughties". This should resolve to that range.

Reply:
Somewhat similarly, "the twenty-tens" should resolve to 2010-2019.

Reply:
"the two thousands" could also be tagged.

Reply:
In many of these cases for the written form the "the" has to be there.

Reply:
Added.